cmd/scollector/collectors/aws.go

```diff
@@ -0,0 +1,98 @@
+package collectors
+
+import (
+	"fmt"
+	"time"
+
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+	"bosun.org/slog"
+
+	"github.com/awslabs/aws-sdk-go/aws"
+	"github.com/awslabs/aws-sdk-go/gen/ec2"
+	//"github.com/awslabs/aws-sdk-go/gen/elb"
+	"github.com/awslabs/aws-sdk-go/gen/cloudwatch"
+)
+
+const (
+	awsCPU string = "aws.ec2.cpu"
+)
+
+func AWS(accessKey, secretKey, region string) {
+	slog.Info("warming up the AWS system")
+	collectors = append(collectors, &IntervalCollector{
+		F: func() (opentsdb.MultiDataPoint, error) {
+			return c_aws(accessKey, secretKey, region)
+		},
+		name: fmt.Sprintf("aws-%s", region),
+	})
+}
+
+func c_aws(accessKey, secretKey, region string) (opentsdb.MultiDataPoint, error) {
+	var md opentsdb.MultiDataPoint
+
+	slog.Info("c_aws called")
+
+	creds := aws.Creds(accessKey, secretKey, "")
+	if creds == nil {
+		slog.Warning("Unable to make creds")
+	}
+	ecc := ec2.New(creds, region, nil)
+	if ecc == nil {
+		slog.Warning("Unable to login to EC2")
+	}
+	//elb := elb.New(creds, region, nil)
+
+	cw := cloudwatch.New(creds, region, nil)
+	if cw == nil {
+		slog.Warning("Unable to login to CloudWatch")
+	}
+	instances, err := AWSGetInstances(*ecc)
+	if err != false {
+		// Do something useful in error
```

Comment:
Besides the bool not being correct, you are being silent here so you could have a problem and not get a message